### PR TITLE
feat: upgrade to Node.js LTS (18)

### PR DESCRIPTION
**Pull request by StarpTech**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by fiam — APPROVED** [review]
I would submit the documentation changes in a separate PR, but let’s get it done. LGTM

File: .github/actions/node/action.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ runs:
   steps:
     - uses: actions/setup-node@v3
       with:
-        node-version: '16'
+        node-version: '18'
```
Comment (thisisnithin) [inline .github/actions/node/action.yaml:21]: Can we do node-version: ${{ inputs.node-version }} instead? the default is already set to LTS.

**Review by thisisnithin — CHANGES_REQUESTED** [review]

File: .github/actions/node/action.yaml (hunk 1 above)
Comment (StarpTech) [reply .github/actions/node/action.yaml]: Good catch. Added.

**Review by Pagebakers — APPROVED** [review]
LGTM

**Review by thisisnithin — APPROVED** [review]
LGTM

**Pull request merged**